jupyter_book/book_template/_config.yml

```diff
@@ -22,8 +22,8 @@ email: choldgraf@berkeley.edu
 description: >- # this means to ignore newlines until "baseurl:"
   This is an example book built with Jupyter Books.
 
-baseurl: "/jupyter-book" # the subpath of your site, e.g. /blog. If there is no subpath for your site, use an empty string ""
```

Comment:
these probably shouldn't be changed in the book_template, since that would change the jupyter book website, no?

Reply:
Totally agree.

Thinking on this, I think it's a more general question of whether we actually want to have a netlify deployment. I went ahead and set one up so we've claimed the domain name, but if this is something we want to keep (the deployment itself) I might set up a new `netlify` branch and build from there.

Reply:
Hmm, I think it'd be fine to have a netlify deployment instead of CircleCI, but I don't want the book's URL to change away from `https://jupyter.org/jupyter-book` since that'd cause confusion for users. Wouldn't this change the URL?

Reply:
ping on this one - I don't think we want to change the jupyter-book URL, will this do that?